### ISSUE TITLE
set_env_attr and get_env_attr not working with current gym release

### ISSUE THREAD

**Issue by seawee1** [aai-institute/tianshou#688]
- [ ] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [ ] RL algorithm bug
    + [x] documentation request (i.e. "X is missing from the documentation.")
    + [ ] new feature request
- [x] I have visited the [source website](https://github.com/thu-ml/tianshou/)
- [x] I have searched through the [issue tracker](https://github.com/thu-ml/tianshou/issues) for duplicates
- [x] I have mentioned version numbers, operating system and environment, where applicable:

For completeness, I struggled to get your env.set_env_attr function to work. I found the error on OpenAI gym's side (see: https://github.com/openai/gym/issues/2952). Maybe you could mention this issue somewhere in the documentation. I use gym  0.24.1.

**Comment by Trinkle23897** [comment]
Could you please provide an example here to show the bug? Thanks!

**Comment by seawee1** [comment]
Sure!

```python
import gym
from tianshou.env import DummyVectorEnv

if __name__ == "__main__":
    def create_env():
        env = gym.make("CartPole-v1")
        print(type(env))  # <class 'gym.wrappers.time_limit.TimeLimit'>
        return env

    train_envs = DummyVectorEnv(
        [create_env]
    )

    train_envs.set_env_attr("test_attribute", 1337)
    print(train_envs.get_env_attr("test_attribute"))  # Returns: [1337]
    print(hasattr(train_envs.workers[0].env, "test_attribute"))  # Returns: True
    print(hasattr(train_envs.workers[0].env.unwrapped, "test_attribute"))  # Returns: False
```

You can see that `train_envs.set_env_attr` sets the attribute inside the wrapper, not the environment.

**Comment by Trinkle23897** [comment]
Fixed in #695

**Comment by seawee1** [comment]
Cool, thanks!
